locale/en/messages.json

```diff
@@ -2927,11 +2943,15 @@
     ]
   },
   "Dashboard": {
-    "translation": "",
+    "translation": "Super Dash",
```

Comment:
❓

Reply:
OOps this was my test. Ill rebase it out. Thanks!